tools/debian/rules

```diff
@@ -43,19 +43,27 @@ override_dh_auto_configure: debian/git-patches-applied
 		--with-pamdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 		--libexecdir=/usr/lib/cockpit $(CONFIG_OPTIONS)
 
-override_dh_auto_install:
-	dh_auto_install
-	mkdir -p debian/tmp/etc/pam.d
-	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
-
 override_dh_systemd_enable:
 	dh_systemd_enable -p cockpit-ws --name=cockpit cockpit.socket
 
 override_dh_systemd_start:
 	dh_systemd_start -p cockpit-ws cockpit.socket
 
 override_dh_install:
-	dh_install --list-missing -Xusr/src/debug
+	# Debian/Ubuntu PAM config
+	mkdir -p debian/tmp/etc/pam.d
+	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
+
+	# don't ship broken branding symlinks
+	for d in rhel fedora centos; do rm -r debian/tmp/usr/share/cockpit/branding/$$d; done
+	dpkg-vendor --derives-from ubuntu || rm -r debian/tmp/usr/share/cockpit/branding/ubuntu
+
+	# unpackaged modules
```

Comment:
> If the dangling symlinks are put into a, say, Debian container, their RHEL logo references still won't magically appear. So I don't see much point in shipping those. If you object heavily, I can mark the bug as "wontfix" and ignore the package check.

They may not appear for a Debian container ... but for Fedora and RHEL they do. The cockpit/ws container explicitly mounts directories from the host when run on Atomic Host.

So I'm fine with making Debian/Ubuntu an exception to the container insanity here. But there's a reason for shipping the branding/dangling links on Fedora, RHEL, CentOS.